Mathlib/GroupTheory/FreeAbelianGroupFinsupp.lean

```diff
@@ -54,53 +54,58 @@ theorem Finsupp.toFreeAbelianGroup_comp_singleAddHom (x : X) :
       (smulAddHom ℤ (FreeAbelianGroup X)).flip (of x) := by
   ext
   simp only [AddMonoidHom.coe_comp, Finsupp.singleAddHom_apply, Function.comp_apply, one_smul,
-    to_free_abelian_group, Finsupp.liftAddHom_apply_single]
+    toFreeAbelianGroup, Finsupp.liftAddHom_apply_single]
 #align finsupp.to_free_abelian_group_comp_single_add_hom Finsupp.toFreeAbelianGroup_comp_singleAddHom
 
 @[simp]
 theorem FreeAbelianGroup.toFinsupp_comp_toFreeAbelianGroup :
     toFinsupp.comp toFreeAbelianGroup = AddMonoidHom.id (X →₀ ℤ) := by
-  ext (x y); simp only [AddMonoidHom.id_comp]
+  apply addHom_ext'
+  intro x
+  apply AddMonoidHom.ext_int
+  apply ext
+  intro y
+  simp only [AddMonoidHom.id_comp]
```

Comment:
This looks like an underlying bug that we should fix. At the very least we should have added a porting note here

Reply:
All guilty `ext` lemmas have been found out and given a high priority.